mpifileutils.spec

```diff
@@ -43,15 +43,15 @@
 %endif
 
 Name:		mpifileutils
-Version:	0.10.1
-Release:	6%{?relval}%{?dist}
+Version:	0.11
+Release:	7%{?relval}%{?dist}
 Summary:	File utilities designed for scalability and performance.
 
 Group:		System Environment/Libraries
 License:	Copyright and BSD License
 URL:		https://hpc.github.io/mpifileutils
 Source:		https://github.com/hpc/%{name}/archive/v%{version}.tar.gz
-Patch1:     0.10.1...1ed76ea.patch
+#Patch1:     0.10.1...1ed76ea.patch
```

Comment:
So this PR is to use upstream's official 0.11 release, without any patch from master?

If so, you can remove https://github.com/daos-stack/mpifileutils-pkg/blob/ab37048e0e0c1417d66ae422c4d11898bc3fa137/Makefile#L4-L6 so that this PR doesn't have a `.g<git-hash>` in the `Release:`.
```suggestion
```

Reply:
We _**will**_ be building against another patch version in the future, since  we are still actively making changes.
In this case, should I still remove the `.g<git-hash>` entirely? Or maybe just comment it out for now?

Reply:
You can leave the `%{?relval}` in `Release:	1%{?relval}%{?dist}` if you want, but you need to remove (or I guess comment out) the the lines in the `Makefile`.  But you should also remove the reference to the `#Patch1:` line and restore it again, when a patch is being used.

Reply:
Sure! Just pushed